_layouts/home.html

```diff
@@ -220,6 +221,50 @@ <h2>Events</h2>
     </div>
 
     <script src="{{site.baseurl}}/assets/js/main.js"></script>
+    <script>
+        $.ajax({
+        method: "get",
+        url: "https://ieee.nitk.ac.in/blog/search.json",
+        success: function (result) {
+        add_blogs(result)
+        }
+        });
+
+        function add_blogs(result) {
+            blog = result;
+            var i = 1;
+            var html_str = '';
+            if (i<=3){
+            for (items in blog ){
+                if (i > 3) { break; }
+
+                str = blog[items].author;
+                str = str.split(" ").join("");
+                author_img_url = `https://ieee.nitk.ac.in/blog/assets/img/authors/${str}.jpeg`;
+
+                html_str += `<div class="card-deck"><div class="animate-box"><article role="article" itemscope="itemscope" itemtype="http://schema.org/BlogPosting" itemprop="blogPost">`;
+                html_str += `<h2><a href="`+ blog[items].url +`">` + blog[items].title + `</a></h2><p class="author-wrap">`;
+                html_str += `<span class="author-img" style="background-image: url(https://ieee.nitk.ac.in/blog/assets/img/authors/${str}.jpeg), url(https://ieee.nitk.ac.in/blog/assets/img/authors/blog-author.jpg);"></span>`;    
+                html_str += `<span class="author">by &nbsp` + blog[items].author + `&nbsp; | &nbsp;` + blog[items].date.split(" ")[1] + `&nbsp` + blog[items].date.split(" ")[0] + `,` + blog[items].date.split(" ")[2] + `</span></p>` ;
+                html_str += `<div class="tags">`;
+
+                var tag = blog[items].tags;
```

Comment:
Can you add the category instead of tags? Quite often we don't get accurate tags from the blog authors and category is a more consistent field.